advanced_functionality/huggingface_deploy_instructpix2pix/deploy-instructpix2pix-to-sagemaker-ir.ipynb

```diff
@@ -0,0 +1,1281 @@
+{
```

Comment:
<p>Do you have legal approval to use this picture</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3954/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
No. Do I need legal approval for HuggingFace models published under MIT license?

Reply:
This is the same picture used in: https://huggingface.co/timbrooks/instruct-pix2pix